scripts/install_third_party_libs.py

```diff
@@ -24,17 +24,23 @@
 
 # These libraries need to be installed before running or importing any script.
 TOOLS_DIR = os.path.join(os.pardir, 'oppia_tools')
-# Download and install pyyaml.
-if not os.path.exists(os.path.join(TOOLS_DIR, 'pyyaml-5.1.2')):
-    subprocess.check_call([
-        sys.executable, '-m', 'pip', 'install', 'pyyaml==5.1.2', '--target',
-        os.path.join(TOOLS_DIR, 'pyyaml-5.1.2')])
-
-# Download and install future.
-if not os.path.exists(os.path.join('third_party', 'future-0.17.1')):
-    subprocess.check_call([
-        sys.executable, '-m', 'pip', 'install', 'future==0.17.1', '--target',
-        os.path.join('third_party', 'future-0.17.1')])
+
+
+PREREQUISITES = [
+    ('pyyaml', '5.1.2', os.path.join(TOOLS_DIR, 'pyyaml-5.1.2')),
+    ('future', '0.17.1', os.path.join('third_party', 'future-0.17.1')),
+]
+
+for PACKAGE, VERSION, PATH in PREREQUISITES:
+    if not os.path.exists(PATH):
+        COMMAND = [
+            sys.executable, '-m', 'pip', 'install', '%s==%s'
+            % (PACKAGE, VERSION), '--target', PATH]
+        UEXTENTION = ['--user', '--prefix=', '--system']
```

Comment:
Ditto

Reply:
Done!

Reply:
ditto

Reply:
I think variables outside a function have to be in this pattern and I couldn't find any workaround other than disabling it so I had to make it capitals.

Reply:
I'm fine if you disable the linter check for all caps only for this for-loop.

Reply:
I am getting a redefined-outer-name that does not get disabled with pylint: disable, how do I disable it?

Reply:
Hmm, even if you do this? https://github.com/jhoerr/box-python-sdk/commit/ae3d2841dbb640063a8bfd626c74a2cc52de55ed#diff-1a087e5c31e69a9634fc854963c1581eR16

Reply:
Yes I did something similar that, but then I did not know exactly why the error was coming then. I figured out after seeing how it was used in link, thanks!